client/scss/components/forms/_drop-zone.scss

```diff
@@ -14,4 +14,9 @@
   &.hovered {
     border-color: $color-teal;
   }
+
+  // override global field padding as fields are centre aligned in upload box
+  .w-field__input {
+    padding-inline-end: 0;
```

Comment:
The improved alignment good; I wonder if it would make more sense to make the original style never apply to the last child, or whether that would cause complications. (I did not look at the markup so it's very possible this is not about being a list child.

Reply:
yeah I think that could be good, except that technically this container can have other fields added to it (bit of a customisation but could be used for custom documents).

I think we should probably keep this generic to the container for any other fields that get added.